### PR TITLE
LPS-192742 Fix ObjectDDMStorageAdapter logic to support Proxy Objects

### PR DESCRIPTION
**Pull request by carolmariaabb**
Related Issue: https://liferay.atlassian.net/browse/LPS-192742

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:test:object

**Comment by carolmariaabb** [comment]
ci:test:object

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
Just started reviewing :)

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_3_2/DDMFormInstanceSettingsUpgradeProcess.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+/**
+ * @author Carolina Barbosa
+ */
+public class DDMFormInstanceSettingsUpgradeProcess extends UpgradeProcess {
+
+	public DDMFormInstanceSettingsUpgradeProcess(JSONFactory jsonFactory) {
+		_jsonFactory = jsonFactory;
+	}
+
```
Comment (marcelabc) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_3_2/DDMFormInstanceSettingsUpgradeProcess.java:27]: Why did you not add @Override ?

File: modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java [hunk 2]
```diff
@@ -81,66 +75,15 @@ public DDMStorageAdapterDeleteResponse delete(
...
-					objectDefinition, objectEntry.getId());
-			}
-
-			return DDMStorageAdapterDeleteResponse.Builder.newBuilder(
-			).build();
-		}
-		catch (Exception exception) {
-			throw new StorageException(exception);
-		}
+		throw new UnsupportedOperationException();
```
Comment (marcelabc) [inline modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java:78]: Why don't we support delete/get operations anymore?

**Review by marcelabc — COMMENTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_3_2/DDMFormInstanceSettingsUpgradeProcess.java (hunk 1 above)
Comment (carolmariaabb) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_3_2/DDMFormInstanceSettingsUpgradeProcess.java:27]: I forgot 😆. I will add it. Thanks!

**Comment by carolmariaabb** [comment]
ci:test:sf

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_3_2/DDMFormInstanceSettingsUpgradeProcess.java (hunk 1 above)
Comment (carolmariaabb) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_3_2/DDMFormInstanceSettingsUpgradeProcess.java:27]: Done! Thanks a lot!

File: modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java (hunk 2 above)
Comment (carolmariaabb) [reply modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java:78]: The Report Entries tab is also hidden

Comment (marcelabc) [reply modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java:78]: Thank you for the details!

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:stable

**Comment by carolmariaabb** [comment]
ci:forward:force

**Comment by carolmariaabb** [comment]
ci:forward:force

**Comment by carolmariaabb** [comment]
Manually forwarded https://github.com/brianchandotcom/liferay-portal/pull/139214

**Comment by carolmariaabb** [comment]
ci:stop

**Pull request closed without merging**